web/src/main/java/ua/com/vertex/logic/UserLogicImpl.java

```diff
@@ -68,9 +67,10 @@ public void saveImage(String email, MultipartFile file, String imageType) {
 
     private void validateMultipartFile(MultipartFile file) {
         if (file.getSize() > fileSizeInBytes) {
-            throw new MultipartValidationException(FILE_SIZE_EXCEEDED);
+            throw new MultipartValidationException("File size exceeded, max allowed is " +
+                    UtilFunctions.humanReadableByteCount(fileSizeInBytes));
         } else if (!file.getContentType().split("/")[0].equals("image")) {
-            throw new MultipartValidationException(FILE_TYPE_INVALID);
+            throw new MultipartValidationException("You have chosen a file of invalid type!");
```

Comment:
can you please be more polite with customer? Without '!' !!!!!!!!!!!!!!!!!!!!!!! =)